packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/components.types.ts

```diff
@@ -0,0 +1,135 @@
+import { FormikProps, FormikErrors } from 'formik';
+import {
+    TCreateStrategy,
+    TDurationUnitDropdown,
+    TTypeStrategiesDropdown,
+    TOnChangeDropdownItem,
+    TOnChangeInputValue,
+    TOnHideDropdownList,
+    TOnScrollStopDropdownList,
+    TSymbolDropdown,
+    TTradeTypeDropdown,
+    TTradeType,
+    TDurationOptions,
+    TSetCurrentFocus,
+    TTypeStrategy,
+    TSetFieldValue,
+    TFormValues,
+    TMarketOption,
+    TGetSizeDesc,
+    TQuickStrategyFormValues,
+    TSelectsFieldNames,
+    TDropdowns,
+    TSelectedValuesSelect,
+    TDropdownItems,
+    TInputBaseFields,
+    TInputsFieldNames,
+} from '../quick-strategy.types';
```

Comment:
@maryia-binary @prince-deriv I think a better approach is to separate types in different files, just like we do with components. Keeping all possible types in one file is rather inconvenient for further support. Of course, if the parent component is small enough and contains few types, why not store them in the same file. But in this case, there is no good reason to overload a file where this can be avoided.